09_maps/main.go

```diff
@@ -221,6 +223,16 @@ func calcBMI(mapInput map[string]int) {
 	// fmt.Println(bmi)
 	// TODO: check and fix syntax for name and bmi variable
 	fmt.Println("BMI is", bmi)
+
+	for {
+		var counter int = 0
+		if counter > len(mapInput) {
+			break
+		}
+		counter++
+		return bmi
+	}
+
```

Comment:
@nobe4 I tried that, gives me `too many return values have (float64) want ()` same if I do `return bmi`.
I tried to just have `return`, that gave me `redundant return statement (S1023)` no matter if with or no loop.

Reply:
Go back to the function exercise and re-read how functions have to be defined.